### PR TITLE
Fix base.html

### LINKED ISSUE #72: Fix and re-organize Base template
The base template is being extended by every template, therefore it serves for different uses and should give freedom of design and remove limitations.

Changes made:
* Added structure - ```<head>``` & ```<body>``` tags
* Added the needed JS links and Bootstrap links (also ajax for future use)
* Removed "Shlifim" prefix from costume title
* Fixed Navbar to be collapsable 
* Added CSS class to static/home/style.css for the navbar color
* More changes below:

Please Notice - I removed the background color definition in the base,
to add background color in your feature create a ```<body>``` tag and add a CSS class with
the wanted color.

Also, I removed the following tags:
 ```<div class="row">```
  ```      <div class="col-md-8">```

They used to warp the block content and thus limited the design possibilities of each
page that extended the base. (Mostly it Interrupted the landing page from being full-screen)
if your feature was changed by this add the tags I mentioned in your template.

"Base.html" is prune to changes in the future in case of:
Adding routes to links in the navbar (replacing ```href=#"```)
Adding more ```<script>``` & ```<link>``` tags 
Prettifying the navbar itself

Let me know if you have any suggestions for improvement or requests for changes.

### PR DESCRIPTION
**Pull request by liornoy**
* Added structure - head - body tags
* Added the needed JS links and bootstrap links (also ajax for future use)
* Removed "Shlifim" prefix from costume title
* Fixed Navbar to be collapsable
* Added CSS class to static/home/style.css for the navbar color

modified:   home/templates/home/base.html
modified:   home/static/home/style.css

fix #72

### CODE REVIEW

**Comment by avivz450** [comment]
LGTM

**Comment by danitLevi** [comment]
Looking good

**Comment by danitLevi** [comment]
looking good

File: home/templates/home/base.html [hunk 1]
```diff
@@ -1,82 +1,79 @@
...
-	integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
-    <link rel="stylesheet" type="text/css" href="{% static 'home/style.css' %}">
+		<!-- JS scripts-->
+		<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/js/bootstrap.bundle.min.js" 
+		integrity="sha384-JEW9xMcG8R+pH31jmWH6WWP0WintQrMb4s7ZOdauHnUtxwoG2vI5DkLtS3qm9Ekf" crossorigin="anonymous"></script>
+		<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+		<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+		
+		<!-- For ajax-->
```
Comment (ifireball) [inline home/templates/home/base.html:16]: Are you including JQuery twice? Why?

File: home/templates/home/base.html [hunk 2]
```diff
@@ -1,82 +1,79 @@
...
 		</div>
-	</div>
-</nav>
-    <main role="main" class="container">
-      <div class="row">
-        <div class="col-md-8">
-          {% block content %}{% endblock %}
-        </div>
-      </div>
-    </main>
```
Comment (ifireball) [inline home/templates/home/base.html:79]: I seem to remember there was an important reason why one of the bootstrap scripts needed to be loaded after the rest of the page was already loaded, so not sure why are you removing this.

**Review by ifireball — CHANGES_REQUESTED** [review]

File: home/templates/home/base.html (hunk 1 above)
Comment (liornoy) [reply home/templates/home/base.html:16]: The first is for the navbar I think, and the second one is for future use for Ajax.

File: home/templates/home/base.html (hunk 2 above)
Comment (liornoy) [reply home/templates/home/base.html:79]: Ok, I moved them back

**Review by ifireball — APPROVED** [review]

**Pull request merged**